Send_message_at_given_time/send_message_at_given_time.py

```diff
@@ -0,0 +1,61 @@
+from selenium import webdriver
+from selenium.common.exceptions import NoSuchElementException
+import datetime
+import time, sys
+# from datetime import time
+
+chrome_browser = webdriver.Chrome(executable_path='G:\Trial_proj\chromedriver')
+chrome_browser.get('https://web.whatsapp.com/')
+
+time.sleep(10)          # Waiting for user to scan the QR code
```

Comment:
1. we need to somehow detect when QR code is scanned because it will be more efficient i.e neither waiting too long nor too short